gradle/plugins/src/main/kotlin/com/saveourtool/save/buildutils/save-cli-download-configuration.gradle.kts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Configuration task to download save-cli once per a project
+ */
+
+package com.saveourtool.save.buildutils
+
+import de.undercouch.gradle.tasks.download.Download
+import org.gradle.kotlin.dsl.*
+
+plugins {
+    id("de.undercouch.download")
+}
+
+tasks.register<Download>("downloadSaveCli") {
+    val saveCliVersion = readSaveCliVersion()
+    val saveCliFileName = saveCliVersion.map { "save-$it-linuxX64.kexe" }
+    val saveCliPath = saveCliVersion.zip(saveCliFileName) { version, fileName ->
```

Comment:
may be additionally log which version do we download?

Reply:
actually, `Download` prints source url, it contains version:
```
> Task :save-backend:downloadSaveCli
Download https://github.com/saveourtool/save-cli/releases/download/v0.3.9/save-0.3.9-linuxX64.kexe
```